rootfs/usr/bin/playtron-factory-reset

```diff
@@ -74,6 +74,7 @@ reset_etc ssh
 reset_etc xdg
 rsync -aHX /usr/etc/gai.conf /etc/gai.conf
 rsync -aHX /usr/etc/sddm.conf /etc/sddm.conf
+rm /etc/playtron-os.conf
```

Comment:
This should have a `-f` to make it non-interactive.

Reply:
Thanks, updated.